src/main/cpp/t06_homework.cpp

```diff
@@ -84,5 +84,24 @@
 using namespace std;
 
 int t06_homework() {
-
+    set <string> slovarb;
+    int n,counter;
+    string hw,sl;
+    sl = "";
+    counter = 0;
+    cin >> n;
+    for (int i = 0; i < n; i++) {
+        cin >> sl;
+        slovarb.insert(sl);
+    };
+    cin >> hw;
+    for (int i = 0; i < hw.size(); i++) {
+        if (hw[i] != ' ') sl += hw[i];
+        else {
+            if (slovarb.find(sl)) counter++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Suspicious condition. The result of find() is an iterator, but it is not properly checked.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4763739)